sagemaker_neo_compilation_jobs/gluoncv_yolo_darknet/train_yolo.py

```diff
@@ -524,22 +432,31 @@ def model_fn(model_dir):
     :param: model_dir The directory where model files are stored.
     :return: a model (in this case a Gluon network)
     """
+    logging.info('Invoking user-defined model_fn')
+
+    import neomxnet
+
+    #check if GPUs area available
+    ctx = mx.gpu()
+
     net = gluon.SymbolBlock.imports(
-        '%s/model-symbol.json' % model_dir,
+        '%s/compiled-symbol.json' % model_dir,
         ['data'],
-        '%s/model-0000.params' % model_dir,
+        '%s/compiled-0000.params' % model_dir,
+        ctx=ctx
     )
+    net.hybridize(static_alloc=True, static_shape=True)
+
+    #run warm-up inference on empty data
+    warmup_data = mx.nd.empty((1,3,320,320), ctx=ctx)
+    class_IDs, scores, bounding_boxes = net(warmup_data)
    
     return net
     
 def transform_fn(net, data, content_type, output_content_type): 
     """
     Transform incoming requests.
     """
-    import os
-
-    os.system('pip3 install gluoncv')
-
```

Comment:
can we log the following at this point:
`logging.info('Invoking user-defined transform_fn')`

Reply:
Done.